hazelcast-client/src/test/java/com/hazelcast/client/quorum/ClientQuorumHelper.java

```diff
@@ -0,0 +1,28 @@
+package com.hazelcast.client.quorum;
+
+import com.hazelcast.client.test.TestHazelcastFactory;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.quorum.PartitionedCluster;
+
+import static com.hazelcast.client.quorum.QuorumTestUtil.createClient;
+
+public class ClientQuorumHelper {
+
+    private TestHazelcastFactory factory;
+    private HazelcastInstance c[];
+    private PartitionedCluster cluster;
+
+    public ClientQuorumHelper(PartitionedCluster cluster, TestHazelcastFactory factory) {
+        this.cluster = cluster;
+        this.c = new HazelcastInstance[cluster.h.length];
```

Comment:
can you give a more descriptive name than c? e.g. cluster?

Reply:
Fixed